phpBB/includes/functions_upload.php

```diff
@@ -332,6 +332,26 @@ function move_file($destination, $overwrite = false, $skip_image_check = false,
 				@unlink($this->destination_file);
 			}
 
+			/**
+			* Event to modify avatars before writing to filesystem
+			*
+			* @event core.avatar_upload
+			* @var  string  filename     filename for upload
+			* @var  string  max_width    max_width board setting
+			* @var  string  max_height   max_height board setting
+			* @var  string  extension    extension of avatar
+			* @since 3.1.11-RC1
+			*/
+			if ($mode === 'avatar')
+			{
+				$filename=$this->filename;
```

Comment:
There should be exactly one space between variables and operators. i.e. it should be `$max_width = $this->upload->max_width`